io.github.ecotubehq.player.json

```diff
@@ -0,0 +1,360 @@
+{
+    "app-id": "io.github.ecotubehq.player",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "45",
+    "sdk": "org.gnome.Sdk",
+    "command": "ecotube",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=all",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-pictures",
+        "--talk-name=org.gtk.vfs.*",
+        "--env=LC_NUMERIC=C"
+    ],
+    "modules": [
+        {
+            "name": "luajit",
+            "no-autogen": true,
+            "cleanup": [ "/bin", "/lib/*.a", "/include", "/lib/pkgconfig", "/share/man" ],
```

Comment:
Add these to a top level cleanup after finish args. No need to repeat them for every module.